docs/reference/errors-and-warnings/NU1604.md

```diff
@@ -32,3 +32,22 @@ or
 > `<PackageReference Version="9.0.0" />`
 
 which implies a lower bound.
+
+If using [Central Package Managment](https://learn.microsoft.com/nuget/consume-packages/Central-Package-Management) you will need to update the `PackageVersion` `Version` attribute in your `Directory.Packages.Props` file.
+For example change from:
+
+> `<PackageVersion Include="PackageA" Version="(9.0.0, )" />`
+
+
+to:
+
+> `<PackageVersion Include="PackageA" Version="[9.0.0, )" />`
+
+or
+
+> `<PackageVersion Include="PackageA" Version="9.0.0" />`
+
+which implies a lower bound.
+
+> [!Note]
+> When using CPM and the file `Directory.Packages.Props` is invalid, NU1604 is raised.
```

Comment:
I added this because the original issue is related to an invalid file but makes me think this is a bug instead of a docs issue. @jeffkl what do you think?